proposal/repository.go

```diff
@@ -66,6 +75,14 @@ func (r *Repository) List(opts repoListOpts) ([]v2.Proposal, error) {
 	} else {
 		q.WriteString(fmt.Sprintf(` AND proposal->'access_policies' @> '[{"id": "%s"}]'`, opts.accessPolicy))
 	}
+	if opts.priceGiBMax > 0 {
+		args = append(args, opts.priceGiBMax)
```

Comment:
IDK if this is more readable, and up til now it's a mix between args and string formatting. SQL Injection protection?